client/analytics/report/customers/config.js

```diff
@@ -167,6 +167,81 @@ export const advancedFilters = {
 				} ) ),
 			},
 		},
+		order_count: {
+			labels: {
+				add: 'No. of Orders',
+				remove: 'Remove order  filter',
+				rule: 'Select an order count filter match',
+				title: 'No. of Orders {{rule /}} {{filter /}}',
+			},
+			rules: [
+				{
+					value: 'max',
+					label: 'Less Than',
+				},
+				{
+					value: 'min',
+					label: 'More Than',
+				},
+				{
+					value: 'between',
+					label: 'Between',
+				},
+			],
+			input: {
+				component: 'Number',
+			},
+		},
+		total_spend: {
+			labels: {
+				add: 'Total Spend',
+				remove: 'Remove total spend filter',
+				rule: 'Select a total spend filter match',
+				title: 'Total Spend {{rule /}} {{filter /}}',
+			},
+			rules: [
+				{
+					value: 'max',
+					label: 'Less Than',
+				},
+				{
+					value: 'min',
+					label: 'More Than',
+				},
+				{
+					value: 'between',
+					label: 'Between',
+				},
+			],
+			input: {
+				component: 'Number',
+			},
+		},
+		avg_order_value: {
+			labels: {
+				add: 'AOV',
```

Comment:
I didn't check the designs, but I thought we were moving away from "AOV" in favor of something more readable.

Reply:
It looks like "AOV" is being used in designs unless I'm looking at outdated ones.  Do you have a link to any discussion on this?  /cc @LevinMedia